Code/Comscore/Source/integration/ComscoreAdapter.swift

```diff
@@ -474,22 +475,18 @@ class THEOComScoreAdapter: NSObject {
     }
     
     private func onLoadedMetadata(event: LoadedMetaDataEvent) {
-        if (comscoreMetadata.length == 0 && !inAd) {
-            player.requestSeekable(completionHandler: { [weak self] seekableRanges, error in
-                if let welf = self,
-                   let foundRanges = seekableRanges?.sorted(by: { range1, range2 in range1.start < range2.start }),
-                   foundRanges.count > 0,
-                   let lastRange = foundRanges.last,
-                   let firstRange = foundRanges.first {
-                    let dvrWindowEnd = lastRange.end
-                    let dvrWindowStart = firstRange.start
-                    let dvrWindowLengthInSeconds = dvrWindowEnd - dvrWindowStart
-                    if (dvrWindowLengthInSeconds > 0) {
-                        if welf.configuration.debug { print("[THEOplayerConnectorComscore] set DVR window length of ",dvrWindowLengthInSeconds) }
-                        welf.streamingAnalytics.setDVRWindowLength(Int(dvrWindowLengthInSeconds*1000))
-                    }
-                }
-            })
+        if self.comscoreMetadata.length != 0 || self.inAd {
```

Comment:
This looks strange to me. Can anyon explain why we return early when comscoreMetadata.length is 0? (same for `onSeeked`)

Reply:
In that event handler the DVR window length should be set if the current source is a live stream. 

-  If  `comscoreMetadata.length == 0` , it's a live stream (see [ComscoreMetadata.length](https://github.com/THEOplayer/iOS-Connector/blob/7b2b9b6d0ce1f9dd20d2daffe5d528215a053eb4/Code/Comscore/Source/integration/ComscoreMetadata.swift#L16)). I assume this is something from an old customer implementation. Doing the check with `comscoreMetadata.mediaType == .live` would make more sense nowadays and is better for readability.
- If `self.inAd == true`, the seekable range for the ad would be returned

Reply:
Thanks, I updated to use `mediaType` instead